stdlib/pervasives.mli

```diff
@@ -1091,6 +1091,15 @@ val at_exit : (unit -> unit) -> unit
    The functions are called in 'last in, first out' order:
    the function most recently added with [at_exit] is called first. *)
 
+(** {6 Misc} *)
+
+val enable_runtime_warnings: bool -> unit
```

Comment:
Agreed to move it to `Sys`.  `Gc` would be weird, since the warning is not directly related to the Gc (esp. if we show it also when the program exits).

`set_runtime_warnings` doesn't sound right, it's not the warnings that one sets, but their enabled state.

One should also discuss if one wants a fine-grained control over each specific warnings if we had more warnings later (in which case one could use a record type, a bitmask, a list of enable warnings, etc), or just a global switch (a `bool` state as in the current proposal).